w05h02/test/pgdp/pingunetwork/behavioral/PinguNetworkTest.java

```diff
@@ -285,6 +285,39 @@ void testGroup4() {
         assertArrayEquals(bob_gang.getMembers(), new User[]{bob, alice});
         assertEquals(bob_gang.getOwner(), bob);
     }
+    
+    @Test
+    @DisplayName("Group - remove owner not part of group, and no user in group")
+    void testGroup5() {
+        User bob = new User("Bob", "i code stuff", pic4);
+        Group bob_gang = new Group("zulip-chat", "more recycling symbols than on most bottles", bob, new Picture("Bayern", new int[][]{{0, 0, 0}, {0, 1, 0}, {0, 0, 0}}));
+        User alice = new User("Alice", "lol", pic2);
+
+        bob_gang.removeUser(bob);
+
+        bob_gang.removeUser(alice);
+        assertArrayEquals(bob_gang.getMembers(), new User[]{});
+        assertEquals(bob_gang.getOwner(), null);
+    }
+    
+    @Test
+    @DisplayName("Group - remove owner, members not empty afterwards, owner arbitrary pos.")
+    void testGroup6() {
+        User bob = new User("Bob", "i code stuff", pic2);
+        Group bob_gang = new Group("zulip-chat", "more recycling symbols than on most bottles", bob, new Picture("Bayern", new int[][]{{0, 0, 0}, {0, 1, 0}, {0, 0, 0}}));
```

Comment:
Would be great if u sticked to the [Java naming conventions](https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html).

Reply:
Pardon? I literally copied the cases from above :D  and just changed the order of actions thus leaving the file consistent with itself. Should have not merge the file initially if you did not like the naming (but that is not from me...)